data-service-client/src/main/java/com/amazonaws/blox/dataservice/client/v1/client/DataServiceLambdaClient.java

```diff
@@ -0,0 +1,9 @@
+package com.amazonaws.blox.dataservice.client.v1.client;
+
+import lombok.AllArgsConstructor;
+
+@AllArgsConstructor
+public class DataServiceLambdaClient {
```

Comment:
In this package, I would expect this to be an interface (i.e. `public interface DataService`), and not an implementation. The implementation could then be in a separate client package.